### DISCUSSION TITLE
About the current support of OsgEarth for OpenGL ES

### DISCUSSION THREAD

**Discussion by DongShuo-96** [pelicanmapping/osgearth#2375] [Support Q&A] [answered]
I would like to know the current support for OpenGL ES by OsgEarth. I have seen the support for GLES and IOS in the update log of OsgEarth 2.4, but I am not sure if OsgEarth still supports smooth operation in GLES based environments. The latest OsgEarth allows for the inclusion and configuration files related to GLES, and if additional configuration is needed. I have found that OSG has gradually implemented support for GLES. Who can help me?

**Comment by gwaldron** [comment, marked as answer]
We do not actively work on GLES or iOS support. It is likely there is work to be done there; but you will need to try it and find out. Please let us know what issues you encounter.

**Reply by DongShuo-96** [reply]
Thank you very much for your answer
